library/src/main/java/com/google/android/exoplayer2/extractor/flv/AudioTagPayloadReader.java

```diff
@@ -58,15 +63,29 @@ public void seek() {
   protected boolean parseHeader(ParsableByteArray data) throws UnsupportedFormatException {
     if (!hasParsedAudioDataHeader) {
       int header = data.readUnsignedByte();
-      int audioFormat = (header >> 4) & 0x0F;
+      audioFormat = (header >> 4) & 0x0F;
       int sampleRateIndex = (header >> 2) & 0x03;
+      int encodingSize = header & 0x01;
       if (sampleRateIndex < 0 || sampleRateIndex >= AUDIO_SAMPLING_RATE_TABLE.length) {
         throw new UnsupportedFormatException("Invalid sample rate index: " + sampleRateIndex);
       }
-      // TODO: Add support for MP3 and PCM.
-      if (audioFormat != AUDIO_FORMAT_AAC) {
-        throw new UnsupportedFormatException("Audio format not supported: " + audioFormat);
+      // TODO: Add support for MP3.
+      if (audioFormat == AUDIO_FORMAT_ALAW || audioFormat == AUDIO_FORMAT_ULAW) {
+
+          String type = (audioFormat == AUDIO_FORMAT_ALAW) ? MimeTypes.AUDIO_ALAW : MimeTypes.AUDIO_ULAW;
+          int encoding = (encodingSize == 1) ? C.ENCODING_PCM_16BIT : C.ENCODING_PCM_8BIT;
+          int minSize = AudioTrack.getMinBufferSize(8000, AudioFormat.CHANNEL_OUT_MONO, encoding);
```

Comment:
As far as I can tell, the only upper limit is the 24 bit size as you pointed out. Will change to NO_VALUE.